labs/lab3-paxos/src/dslabs/paxos/PaxosServer.java

```diff
@@ -54,13 +61,16 @@ public PaxosLogSlotStatus status(int logSlotNum) {
 
     /**
      * Return the command associated with a given slot in the server's local
-     * log. If the slot has status {@link PaxosLogSlotStatus#CLEARED} or {@link
-     * PaxosLogSlotStatus#EMPTY}, this method should return {@code null}. If
-     * clients wrapped commands in {@link dslabs.atmostonce.AMOCommand}, this
+     * log. This method should return {@code null} if and only if the slot has
```

Comment:
This comment is not equivalent to the previous text.

Reply:
The change was intentional, but based on what you said above regarding the additional invariant it sounds like the open-ended "null" return was as designed and I misunderstood. I'll try to incorporate this intent another way.